packages/honeycomb-layout/package.json

```diff
@@ -8,8 +8,8 @@
     "clean": "rimraf pids logs coverage dist",
     "lint": "eslint --format=node_modules/eslint-formatter-pretty .",
     "start": "npm run start:production",
-    "start:development": "pm2-dev start .config/pm2.development.json",
-    "start:production": "pm2 start .config/pm2.production.json",
+    "start:development": "cross-env NODE_ENV=development nodemon src/index.js --exec babel-node --watch src/ --ext js",
+    "start:production": "cross-env NODE_ENV=production node dist/index.js",
```

Comment:
i think we have following options:
* make production everywhere as default 
* set NODE_ENV to production manual via cli (we set it already in docker-files)
* move it from devDependencies to dependencies, but i think we should keep the node_modules in production as small as possible. what do you think?